src/api/spec/policies/project_policy_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe ProjectPolicy do
+  let(:anonymous_user) { create(:user_nobody) }
+  let(:user) { create(:confirmed_user) }
+  let(:other_user) { create(:confirmed_user) }
+  let(:admin_user) { create(:admin_user) }
+  let(:project) { user.home_project }
+  subject { ProjectPolicy }
+
+  permissions :create?, :can_create_package_in?, :unlock?, :local_project_and_allowed_to_create_package_in? do
```

Comment:
local_project_and_allowed_to_create_package_in? is private, no need to test them